bidwire/alembic/versions/bd1182b81146_added_column_categories.py

```diff
@@ -0,0 +1,28 @@
+"""added column categories
+
+Revision ID: bd1182b81146
+Revises: b73811be5f44
+Create Date: 2017-04-23 06:01:50.631106
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'bd1182b81146'
+down_revision = 'b73811be5f44'
```

Comment:
This down revision is pointing at `b73811be5f44_add_all_basic_bid_fields.py`, which is not the most recent database schema revision. Instead, it should be pointing at the latest revision before yours.

I believe this is also the reason why this PR is not passing the continuous build right now. From the logs on https://travis-ci.org/RagtagOpen/bidwire/builds/224941801 :
```
ERROR [alembic.util.messaging] Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads
FAILED: Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads
```